packages/ui/src/mintlify/code/CodeGroup.tsx

```
@@ -0,0 +1,112 @@
+    console.warn(
+      "CodeGroup has no children, expected at least one CodeBlock child.",
+    );
+    return null;
+  } else if (!Array.isArray(children)) {
+    // Allow looping over a single child
+    children = [children];
+  } else if (children.length === 0) {
+    return null;
+  }
+
+  const [activeTab, setActiveTab] = useState("0");
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)